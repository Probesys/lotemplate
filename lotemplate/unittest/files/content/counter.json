{
  "solutions": {
    "type": "array",
    "value": [
      {
        "title": "Title 1",
        "paragraph": "Content no 1"
      },
      {
        "title": "Title 2",
        "paragraph": "Content no 2"
      },
      {
        "title": "Title 3",
        "paragraph": "Content no 3"
      }
    ]
  }
}