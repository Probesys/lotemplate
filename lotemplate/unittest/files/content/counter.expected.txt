
Basic counter



chapter 1 : Introduction

Text of the introduction

chapter 2 : Context

A ) political context

lorem ipsum

B ) geographical context

lorem ipsum 2

chapter 3 : Solutions




A ) Title 1

Content no 1

B ) Title 2

Content no 2

C ) Title 3

Content no 3


chapter 4 : Conclusion

Lorem ipsum 


Hierarchical counter



chapter 1 : Context

1.a ) political context

lorem ipsum

1.b ) geographical context

lorem ipsum 2

chapter 2 : Context

2.a ) political context

lorem ipsum

2.b ) geographical context

lorem ipsum 2

Hidden counter and element counter.format




a ) Title 1

Content no 1


b ) Title 2

Content no 2


c ) Title 3

Content no 3



we displayed 3 solutions
